web/controllers/character_controller.ex

```diff
@@ -0,0 +1,60 @@
+defmodule Yggdrasil.CharacterController do
```

Comment:
Only thing I don't have yet is a patch handler for updates per JSONAPI's recommendation. Eventually will get there.